connectors/v2/youtube.js

```diff
@@ -51,12 +51,48 @@ Connector.getArtistTrack = function () {
 		track = text.substr(separator.index + separator.length);
 	}
 
-	/**
-	* Clean non-informative garbage from title
-	*/
+	return {
+		artist: cleanseArtist(artist),
+		track: cleanseTrack(track)
+	};
+};
+
+Connector.getPlaylist = function() {
+	var playlist = [];
+	var $container = $('#eow-description');
+
+	// for each line
+	var potentialTracks = $container.html().split(/\r\n|\r|\n|<br>/g);
+
+	_.each(potentialTracks.reverse(), function(maybeTrack) {
+		var $maybeTrack = $('<div>'+maybeTrack+'</div>');
+		var $timestamp = $maybeTrack.find('a[href=\'#\'][onclick]');
+
+		if($timestamp.length) {
+			var entry = {};
+			entry.startTime = Connector.stringToSeconds($timestamp.first().text());
+			$timestamp.remove();
```

Comment:
Do more to remove timestamp delimiters.

* **E.g. 1:** `00:00 - Trackname` -> Should remove the ` - ` (and other similar delimiters)
* **E.g. 2:** `Trackname [00:00]` -> Should remove the `[` and `]` (and other similar delimiters)